eo-runtime/src/main/java/org/eolang/Universe.java

```diff
@@ -76,9 +76,30 @@ public Universe() {
      * @return Vertex of the object to find.
      */
     public int find(final String name) {
-        Phi accum = this.connector;
+        if (name == null) {
+            throw new IllegalArgumentException(
+                "Argument name is null"
+            );
+        }
+        Phi accum;
         final String[] atts = Universe.replace(name)
             .split("\\.");
+        if ("Q".equals(atts[0])) {
+            accum = Phi.Φ;
+            atts[0] = "";
+        } else if ("ρ".equals(atts[0])) {
+            accum = this.connector;
+        } else if ("$".equals(atts[0])) {
+            accum = this.connector;
+            atts[0] = "";
+        } else {
+            throw new ExFailure(
+                String.format(
+                    "Universe.find starts with %s, but it should start with Q or ^ or $ only",
```

Comment:
@levBagryansky should be only `Q` or `$`, nothing else. If `\rho` is needed, it should be `$.^`